vendor/engines/secure_rooms/app/services/secure_rooms/access_handlers/order_handler.rb

```diff
@@ -15,14 +15,16 @@ def initialize(occupancy)
       end
 
       def process
-        return unless occupancy.account_id?
+        return if occupancy.skip_order?
```

Comment:
I think you're trying to account for the case of swipe-out with no swipe in (because we'll have no order at that point), but I'm not super crazy about adding a database field, especially at the event level for this. Is there another way we can accomplish this?